nomad/acl_endpoint.go

```diff
@@ -578,12 +579,14 @@ func (a *ACL) DeleteTokens(args *structs.ACLTokenDeleteRequest, reply *structs.G
 	// Determine if we are deleting local or global tokens
 	hasGlobal := false
 	allGlobal := true
+	nonexistentTokens := make([]string, 0)
 	for _, accessor := range args.AccessorIDs {
 		token, err := state.ACLTokenByAccessorID(nil, accessor)
 		if err != nil {
 			return fmt.Errorf("token lookup failed: %v", err)
 		}
-		if token == nil {
+		if token == nil { // why continue if the token is nil???
```

Comment:
Seems like you can remove this comment as it appears the reason we continue is to return a complete list of missing tokens (which is nice from a UX perspective).

Reply:
Yes, thank you! That is leftover from development notes, it shouldn't be merged